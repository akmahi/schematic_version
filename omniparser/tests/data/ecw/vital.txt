apuid|RecordID|ItemID|PatientId|PatientAccountNo|EncounterId|ProviderNPI|VitalsCapturedBy|VitalsName|StandardVital|VitalsValue|ModifiedDate
305922|305922.545690.2459365.215|215|545690|545690|2459365|1659584084|9223|Ht|Ht|70"|2022-10-17
305922|305922.545690.2459365.199654|199654|545690|545690|2459365|1659584084|9223|Pain scale||0|2022-10-17
305922|305922.12828.2546997.211|211|12828|516952|2546997|1467551002|542026|Temp|Temp|98.4|2022-10-17
305922|305922.12828.2546997.212|212|12828|516952|2546997|1467551002|542026|BP|BP|118/68|2022-10-17
305922|305922.12828.2546997.213|213|12828|516952|2546997|1467551002|542026|HR||68|2022-10-17
305922|305922.12828.2546997.214|214|12828|516952|2546997|1467551002|542026|RR|RR|16|2022-10-17
305922|305922.12828.2546997.216|216|12828|516952|2546997|1467551002|542026|Wt|Wt|125|2022-10-17
305922|305922.531654.2525555.215|215|531654|531654|2525555|1417179151|9223|Ht|Ht|68.4|2022-10-12
305922|305922.531654.2525555.199654|199654|531654|531654|2525555|1417179151|9223|Pain scale||0|2022-10-12
305922|305922.15774.2531215.211|211|15774|102088|2531215|1518170364|529704|Temp|Temp|97.2|2022-10-17
305922|305922.15774.2531215.212|212|15774|102088|2531215|1518170364|529704|BP|BP|118/76|2022-10-17
305922|305922.15774.2531215.214|214|15774|102088|2531215|1518170364|529704|RR|RR|17|2022-10-17
305922|305922.15774.2531215.215|215|15774|102088|2531215|1518170364|529704|Ht|Ht|50"|2022-10-17
305922|305922.15774.2531215.216|216|15774|102088|2531215|1518170364|529704|Wt|Wt|72.6|2022-10-17
305922|305922.15774.2531215.170669|170669|15774|102088|2531215|1518170364|529704|BMI|BMI|20.42|2022-10-17
305922|305922.15774.2531215.194603|194603|15774|102088|2531215|1518170364|529704|Oxygen sat %||98|2022-10-17
305922|305922.15774.2531215.195876|195876|15774|102088|2531215|1518170364|529704|Pulse sitting|PL|70|2022-10-17
305922|305922.13711.2533754.215|215|13711|520912|2533754|1518170364|539166|Ht|Ht|70 in|2022-10-17
305922|305922.13711.2533754.216|216|13711|520912|2533754|1518170364|539166|Wt|Wt|297|2022-10-17
305922|305922.13711.2533754.170669|170669|13711|520912|2533754|1518170364|539166|BMI|BMI|42.61|2022-10-17
305922|305922.11212.2534007.215|215|11212|505284|2534007|1518170364|539166|Ht|Ht|65|2022-10-17
305922|305922.11212.2534007.216|216|11212|505284|2534007|1518170364|539166|Wt|Wt|175|2022-10-17
305922|305922.11212.2534007.170669|170669|11212|505284|2534007|1518170364|539166|BMI|BMI|29.12|2022-10-17
305922|305922.534136.2526658.215|215|534136|534136|2526658|1518170364|539166|Ht|Ht|65"|2022-10-17
305922|305922.534136.2526658.216|216|534136|534136|2526658|1518170364|539166|Wt|Wt|217.8|2022-10-17
305922|305922.534136.2526658.170669|170669|534136|534136|2526658|1518170364|539166|BMI|BMI|36.24|2022-10-17
305922|305922.11878.2529855.215|215|11878|510541|2529855|1518170364|539166|Ht|Ht|66.5|2022-10-17
305922|305922.11878.2529855.216|216|11878|510541|2529855|1518170364|539166|Wt|Wt|203|2022-10-17
305922|305922.11878.2529855.170669|170669|11878|510541|2529855|1518170364|539166|BMI|BMI|32.27|2022-10-17
