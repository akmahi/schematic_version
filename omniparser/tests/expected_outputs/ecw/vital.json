[
	{
		"vital": {
			"captured_by": "9223",
			"encounter": {
				"id": "2459365"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "545690",
				"id": "545690"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1659584084"
			},
			"record_id": "305922.545690.2459365.215",
			"standard_vital": "Ht",
			"value": "70'"
		}
	},
	{
		"vital": {
			"captured_by": "9223",
			"encounter": {
				"id": "2459365"
			},
			"item_id": "199654",
			"modified_date": "2022-10-17",
			"name": "Pain scale",
			"patient": {
				"account_no": "545690",
				"id": "545690"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1659584084"
			},
			"record_id": "305922.545690.2459365.199654",
			"value": "0"
		}
	},
	{
		"vital": {
			"captured_by": "542026",
			"encounter": {
				"id": "2546997"
			},
			"item_id": "211",
			"modified_date": "2022-10-17",
			"name": "Temp",
			"patient": {
				"account_no": "516952",
				"id": "12828"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.12828.2546997.211",
			"standard_vital": "Temp",
			"value": "98.4"
		}
	},
	{
		"vital": {
			"captured_by": "542026",
			"encounter": {
				"id": "2546997"
			},
			"item_id": "212",
			"modified_date": "2022-10-17",
			"name": "BP",
			"patient": {
				"account_no": "516952",
				"id": "12828"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.12828.2546997.212",
			"standard_vital": "BP",
			"value": "118/68"
		}
	},
	{
		"vital": {
			"captured_by": "542026",
			"encounter": {
				"id": "2546997"
			},
			"item_id": "213",
			"modified_date": "2022-10-17",
			"name": "HR",
			"patient": {
				"account_no": "516952",
				"id": "12828"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.12828.2546997.213",
			"value": "68"
		}
	},
	{
		"vital": {
			"captured_by": "542026",
			"encounter": {
				"id": "2546997"
			},
			"item_id": "214",
			"modified_date": "2022-10-17",
			"name": "RR",
			"patient": {
				"account_no": "516952",
				"id": "12828"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.12828.2546997.214",
			"standard_vital": "RR",
			"value": "16"
		}
	},
	{
		"vital": {
			"captured_by": "542026",
			"encounter": {
				"id": "2546997"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "516952",
				"id": "12828"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.12828.2546997.216",
			"standard_vital": "Wt",
			"value": "125"
		}
	},
	{
		"vital": {
			"captured_by": "9223",
			"encounter": {
				"id": "2525555"
			},
			"item_id": "215",
			"modified_date": "2022-10-12",
			"name": "Ht",
			"patient": {
				"account_no": "531654",
				"id": "531654"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1417179151"
			},
			"record_id": "305922.531654.2525555.215",
			"standard_vital": "Ht",
			"value": "68.4"
		}
	},
	{
		"vital": {
			"captured_by": "9223",
			"encounter": {
				"id": "2525555"
			},
			"item_id": "199654",
			"modified_date": "2022-10-12",
			"name": "Pain scale",
			"patient": {
				"account_no": "531654",
				"id": "531654"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1417179151"
			},
			"record_id": "305922.531654.2525555.199654",
			"value": "0"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "211",
			"modified_date": "2022-10-17",
			"name": "Temp",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.211",
			"standard_vital": "Temp",
			"value": "97.2"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "212",
			"modified_date": "2022-10-17",
			"name": "BP",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.212",
			"standard_vital": "BP",
			"value": "118/76"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "214",
			"modified_date": "2022-10-17",
			"name": "RR",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.214",
			"standard_vital": "RR",
			"value": "17"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.215",
			"standard_vital": "Ht",
			"value": "50'"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.216",
			"standard_vital": "Wt",
			"value": "72.6"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "170669",
			"modified_date": "2022-10-17",
			"name": "BMI",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.170669",
			"standard_vital": "BMI",
			"value": "20.42"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "194603",
			"modified_date": "2022-10-17",
			"name": "Oxygen sat %",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.194603",
			"value": "98"
		}
	},
	{
		"vital": {
			"captured_by": "529704",
			"encounter": {
				"id": "2531215"
			},
			"item_id": "195876",
			"modified_date": "2022-10-17",
			"name": "Pulse sitting",
			"patient": {
				"account_no": "102088",
				"id": "15774"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.15774.2531215.195876",
			"standard_vital": "PL",
			"value": "70"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2533754"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "520912",
				"id": "13711"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.13711.2533754.215",
			"standard_vital": "Ht",
			"value": "70 in"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2533754"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "520912",
				"id": "13711"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.13711.2533754.216",
			"standard_vital": "Wt",
			"value": "297"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2533754"
			},
			"item_id": "170669",
			"modified_date": "2022-10-17",
			"name": "BMI",
			"patient": {
				"account_no": "520912",
				"id": "13711"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.13711.2533754.170669",
			"standard_vital": "BMI",
			"value": "42.61"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2534007"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "505284",
				"id": "11212"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11212.2534007.215",
			"standard_vital": "Ht",
			"value": "65"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2534007"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "505284",
				"id": "11212"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11212.2534007.216",
			"standard_vital": "Wt",
			"value": "175"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2534007"
			},
			"item_id": "170669",
			"modified_date": "2022-10-17",
			"name": "BMI",
			"patient": {
				"account_no": "505284",
				"id": "11212"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11212.2534007.170669",
			"standard_vital": "BMI",
			"value": "29.12"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2526658"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "534136",
				"id": "534136"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.534136.2526658.215",
			"standard_vital": "Ht",
			"value": "65'"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2526658"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "534136",
				"id": "534136"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.534136.2526658.216",
			"standard_vital": "Wt",
			"value": "217.8"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2526658"
			},
			"item_id": "170669",
			"modified_date": "2022-10-17",
			"name": "BMI",
			"patient": {
				"account_no": "534136",
				"id": "534136"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.534136.2526658.170669",
			"standard_vital": "BMI",
			"value": "36.24"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2529855"
			},
			"item_id": "215",
			"modified_date": "2022-10-17",
			"name": "Ht",
			"patient": {
				"account_no": "510541",
				"id": "11878"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11878.2529855.215",
			"standard_vital": "Ht",
			"value": "66.5"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2529855"
			},
			"item_id": "216",
			"modified_date": "2022-10-17",
			"name": "Wt",
			"patient": {
				"account_no": "510541",
				"id": "11878"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11878.2529855.216",
			"standard_vital": "Wt",
			"value": "203"
		}
	},
	{
		"vital": {
			"captured_by": "539166",
			"encounter": {
				"id": "2529855"
			},
			"item_id": "170669",
			"modified_date": "2022-10-17",
			"name": "BMI",
			"patient": {
				"account_no": "510541",
				"id": "11878"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"record_id": "305922.11878.2529855.170669",
			"standard_vital": "BMI",
			"value": "32.27"
		}
	}
]
